core/node/block_reverter/src/lib.rs

```diff
@@ -380,6 +380,8 @@ impl BlockReverter {
         object_store: &dyn ObjectStore,
         deleted_snapshots: &[SnapshotMetadata],
     ) -> anyhow::Result<()> {
+        const CONCURRENT_REMOVE_REQUESTS: usize = 20;
```

Comment:
Is this a reasonable value? Unfortunately, I haven't managed to find "native" batching for GCS API (it looks like classic REST API, so it's not quite unexpected).

Reply:
I think it's reasonable